model/registry/deprecated/container.yaml

```diff
@@ -9,3 +9,23 @@ groups:
         brief: "Deprecated, use `container.label` instead."
         stability: experimental
         deprecated: "Replaced by `container.label`."
+      - id: container.cpu.state
+        brief: "The CPU state for this data point."
+        type:
+          allow_custom_values: true
+          members:
+            - id: user
+              value: 'user'
+              brief: "When tasks of the cgroup are in user mode (Linux). When all container processes are in user mode (Windows)."
```

Comment:
btw we are missing these briefs now in actual attributes. You can add metric specific brief additionally to the ref

Reply:
Well, these are the briefs of the specific members not the attributes.
If we want to provide specific brief per metric family we would need to redefine the whole attribute's members I guess? 
That sounds weird to me, that's why I propose to completely merge them and make the descriptions generic.
@open-telemetry/semconv-system-approvers what do you think about this?

Reply:
I understand, you are correct we can't currently redefine specific members of enums

Reply:
Keeping this open as I would love to get feedback from @open-telemetry/semconv-system-approvers here.
If we really want to provide specific descriptions for the enum's members per namespace then maybe this merging we are attempting here does not really make sense 🤔 ?

Reply:
To me the deciding factor here is whether the different descriptions are clarifications or if they are truly different things. At a first glance, it sounds to me like they are clarifications

Reply:
@braydonk @jamesmoessis any thoughts here?

Reply:
We discussed about it during the last System Metrics WG (May 30th).

What we conclude on is that we would like to have the option to override the briefs and only use a subset of the defined members.
In that case the merging makes sense.

_______
I see that https://github.com/open-telemetry/build-tools/issues/192 would potentially solve this.
So I wonder if we should wait for it then. 

/cc @trisch-me @lmolkova

Reply:
Sounds great! 

let's not wait - we can override briefs already and provide extended list of possible states/modes in notes.

We do this for `error.type` in other places. 

I believe it wouldn't even be a breaking change to set the type to a string and then change it to the enum (code generation would generate string enum as string attribute anyway), but an alternative can look like

```yaml
id: cpu.state|mode
type: 
  members:
    -id: user
    # maybe other if they're common 
    # if even `user` is controversial, use `_OTHER`, `other`, `unknown` etc...
```

# either way, it could be overridden like 
```yaml
attributes:
  ref: cpu.state|mode
  brief: updated brief
  # TODO: fix when https://github.com/open-telemetry/build-tools/issues/192 is implemented
  note: "Following states|modes SHOULD be used `system`, `user`, `idle`, etc"
```

Reply:
thanks @lmolkova for examples and @ChrsMark for implementing it, the issue is resolved from my POV.

Reply:
Thank's everyone for the feedback on this. Resolving this.